src/v3/pages/reports/meeting_attendance/components/report_results/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useEffect, useState } from 'react';
+import { Typography, Box } from '@mui/material';
+import { CardHeader, ScrollableTabs } from '@components/index';
+import {
+  StyledDivider,
+  StyledColumnBox,
+  StyledContentBox,
+  StyledContentCardBox,
+  StyledRowBox,
+  StyledHeaderBox,
+} from './index.styles';
+
+import { TFunction } from 'i18next';
+import {
+  getTeochraticalYear,
+  getTheocraticalMonthDate,
+  getTheocraticalMonthListInAYear,
+  getTheocraticalYearsList,
+} from '@utils/date';
+import { MeetingAttendanceType } from '@definition/meeting_attendance';
+
+const NUMBER_OF_YEARS = 5;
+
+const ReportResultElement = ({ title, value }: { title: string; value: string }) => {
+  return (
+    <StyledRowBox>
+      <StyledRowBox sx={{ width: '100%' }}>
+        <Typography className="body-regular" color="var(--black)">
+          {title}
+        </Typography>
+        <Typography className="h4" color="var(--black)">
+          {value}
+        </Typography>
+      </StyledRowBox>
+    </StyledRowBox>
+  );
+};
+
+const ReportHistoryMonthBox = ({
+  title,
+  t,
```

Comment:
@Logan08691: for consistency, just use the `t` function from the `useAppTranslation` hook on this file. Thank you.